Signal/src/environment/PropertyListPreferences.h

```diff
@@ -61,8 +61,8 @@ extern NSString *const PropertyListPreferencesKeyEnableDebugLog;
 
 - (TSImageQuality)imageUploadQuality;
 
-- (nullable NSString *)lastRanVersion;
-- (NSString *)setAndGetCurrentVersion;
++ (nullable NSString *)lastRanVersion;
++ (NSString *)setAndGetCurrentVersion;
```

Comment:
I went ahead and converted all NSUserDefaults-backed prefs to be class methods, as another measure to future-proof us against these kinds of mistakes.